API/.env.template

```
@@ -6,12 +6,11 @@ DB_NAME_RUN=llio
 DB_NAME_TEST=llio_test
 AZUREAD_CLIENT_ID=app_client_id
 AZUREAD_CLIENT_SECRET=app_client_secret 
-AZUREAD_TENANT_ID=app_tentant_id
 AZUREAD_CALLBACK_URL=http://localhost:8080/auth/azureadv2/callback
 ENV=DEV
 SESSION_MAX_AGE=86400
 SESSION_SECRET=secret
-FRONTEND_URL=http://localhost:5173/
+FRONTEND_URL=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FRONTEND_URL key should go before the SESSION_MAX_AGE key

(UnorderedKey)